client/layout/controller.js

```diff
@@ -141,6 +128,20 @@ export const getPages = () => {
 		} );
 	}
 
+	pages.push( {
+		container: AnalyticsReport,
+		path: '/customers',
+		breadcrumbs: [
+			...initialBreadcrumbs,
+			__( 'Customers', 'woocommerce-admin' ),
+		],
+		wpOpenMenu: 'toplevel_page_woocommerce',
+		navArgs: {
+			id: 'woocommerce-analytics-customers',
+		},
+		capability: 'view_woocommerce_reports',
+	} );
```

Comment:
This change should be reverted, no? The Customers menu item isn't rendered in the legacy menu, but is in the new navigation - where it leads to a broken Customers report (when the analytics feature is disabled).